src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,23 @@
+.footer {
+   background-color: #eaeaea;
```

Comment:
there is no need to add a dot (.) before footer. footer in itself is a semantic it can be directly accessed in css

Reply:
removed footer class and added styles to semantic footer tag